packages/json-api/src/-private/cache.ts

```diff
@@ -405,13 +416,43 @@ export default class JSONAPICache implements Cache {
         console.log(`EmberData | Operation - patch ${op.op}`, op);
```

Comment:
we should take the opportunity to update to the new logging syntax for operations